src/settings/dualKpiChartSettings.ts

```diff
@@ -24,9 +24,7 @@
  *  THE SOFTWARE.
  */
 
-namespace powerbi.extensibility.visual {
-    export class DualKpiChartSettings {
-        public topChartType: string = "area";
-        public bottomChartType: string = "area";
-    }
-}
\ No newline at end of file
+export class DualKpiChartSettings {
+    public topChartType: string = "area";
+    public bottomChartType: string = "area";
```

Comment:
(minor) we should use Enum here instead of string

Reply:
Done!